Remove references to SCA Part 2 (#2909)
theforeman/foreman-documentation: guides/common/modules/proc_setting-the-service-level.adoc

[id="Setting_the_Service_Level_{context}"]
= Setting the service level

> You can configure an activation key to define a default service level for the new host assigned various attributes during registration.

Reviewer: Even though subscription attachments are no longer involved, service level (and other syspurpose attributes) still affect availability of repositories on hosts.

```suggestion
You can configure an activation key to define a default service level for the new host created with the activation key.
Setting a default service level makes specific repositories available to the host.
For example, if the default service level on an activation key is set to Premium, only repositories with premium service levels are available to the host upon registration.
```

[id="Setting_the_Service_Level_{context}"]
= Setting the service level

> You can configure an activation key to define a default service level for the new host created with it.

Reviewer 2: ```suggestion
You can configure an activation key to define a default service level for the new host created with the activation key.
```
I recommend to revert this change. Otherwise, it might not clear from a quick read what "it" refers to (activation key? or service level? or host?).

Setting a default service level makes specific repositories available to the host.

> For example, if the default service level on an activation key is set to Premium, only repositories with premium service levels are available to the host after registration.

Reviewer: I'm pretty sure repositories without a service level set will also be available. (It's only repos with non-matching service levels that would be hidden.)

```suggestion
For example, if the default service level on an activation key is set to Premium, repositories with premium service levels are available to the host after registration.
```
